appcues/src/debug/java/com/appcues/CompositionTestHelper.kt

```diff
@@ -125,6 +127,76 @@ public fun ComposeContainer(context: Context, stepContentJson: List<String>?, tr
     }
 }
 
+// This helper supports testing any experience JSON content, rendering the given group and step index.
+@Composable
+public fun ComposeContainer(
```

Comment:
this is _mostly_ like the trait version, but it handles some special cases for experience tests